packages/core/addon/models/bard-request-v2/request.ts

```diff
@@ -200,7 +200,7 @@ export default class RequestFragment extends Fragment.extend(Validations) implem
    */
   @computed('filters.[]')
   get dateTimeFilter() {
-    return this.filters.find(filter => filter.type === 'timeDimension' && filter.field === 'dateTime');
+    return this.filters.filter(filter => filter.type === 'timeDimension')[0];
```

Comment:
Can we make an issue to visit these date/grain properties?

Reply:
https://github.com/yahoo/navi/issues/1021